### ISSUE TITLE
doesPathContain() produces wrong results

### ISSUE THREAD

**Issue by c-f-h** [gruntjs/grunt#1477]
This is node v5.7.0, grunt v0.4.5.

```
> var grunt = require('grunt');
undefined
> grunt.file.doesPathContain('/home', '/home/foo')
true
> grunt.file.doesPathContain('/home/foo', '/home/foo/bar')
false
```

What am I missing here?

**Comment by c-f-h** [comment]
Here's some more digging:

```
> path.relative('/home/foo/bar', '/home/foo')
'../../foo'
> path.relative('/home/foo', '/home')
'..'
```

**Comment by c-f-h** [comment]
Guess this is not the fault of you guys. Escalated it:

https://github.com/nodejs/node/issues/5489

**Comment by shama** [comment]
This has been fixed in node `5.7.1` and will be released shortly.
